config/github-webhook/verify-signed-off.rb

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env ruby
+
+#
+# Simple(ish) Ruby Sinatra bot to verify that all commits on a Github
+# pull request have a Signed-off-by line in the commit message.
+#
+# Note that this code was heavily inspired by
+# http://git-scm.com/book/en/v2/GitHub-Scripting-GitHub
+#
+##################################################################################
+
+# Yum-installed gems; apparently CentOS doesn't automatically add them
+# to the ruby search path.  :-( There's probably a better way to do
+# this, but I'm not One with the Ruby...
+$LOAD_PATH.unshift '/usr/lib/ruby/gems/1.8/gems/crack-0.1.8/lib'
```

Comment:
`json` does not appear in the output of `gem list`, so something must have gone wonky when installing it:
```
*** LOCAL GEMS ***

crack (0.1.8)
daemons (1.0.10)
eventmachine (0.12.10)
httparty (0.6.1)
rack (1.1.0)
rack-test (0.5.4)
sinatra (1.0)
thin (1.2.8)
```

Maybe someone with sudo privileges should try re-installing the gem.